src/Morphir/Visual/Components/SectionComponent.elm

```diff
@@ -27,15 +28,19 @@ view theme config =
                     else
                         theme.icons.closed
             in
-            row
-                [ width fill
-                , Background.color theme.colors.lightest
-                , Font.bold
-                , smallSpacing theme |> spacing
-                , onClick config.onToggle
-                , pointer
-                ]
-                [ el [] (text icon), el [ Font.size theme.fontSize ] (text config.title) ]
+            Element.Input.button
```

Comment:
Changing clickable elements (divs) to buttons makes them focusable by tab without additional changes, you can activate them by pressing enter, and it's supposed to be better from an accessibility perspective (for screen readers etc.)